tests/test_parser.py

```diff
@@ -1509,3 +1509,11 @@ def test_escape_directive_ignore_after_comment(self, dfparser):
                 'value': 'touch foo;     touch bar'
             }
         ]
+
+    def test_alt_dockerfile_names(self, tmpdir):
+        content = "FROM fedora:38"
+        out = DockerfileParser(path=tmpdir, dockerfile_filename="Containerfile")
+        out.content = content
+
+        validate = DockerfileParser(path=tmpdir, dockerfile_filename="Containerfile")
+        assert validate.baseimage == out.baseimage
```

Comment:
Could this also test the existence of `tmpdir / "Containerfile"`? Technically, this test would pass even if dockerfile_filename didn't do anything